delphi/GrFN/linking.py

```diff
@@ -74,7 +74,7 @@ def get_call_conventions(GrFN):
 
 
 def output_link_graph(G):
```

Comment:
@pauldhein suggested change: add an optional argument called `filename`, with the default value being `linking_graph.pdf`, and replace `"linking-graph2.pdf"` on line 78 with `filename`.

Reply:
Good point, I've updated the code.